sass/_imports.sass

```diff
@@ -10,7 +10,7 @@ $prefix-for-opera: false
 // Incuna-sass
 @import incuna-sass/functions
 @import incuna-sass/normalize
-// @import incuna-sass/reset
+@import incuna-sass/incuna-normalize
```

Comment:
aren't we always using both reset and normalize? 

and what's incuna-normalize?